### ISSUE TITLE
Error while installing latest development version - namespace ‘dbplyr’ 0.0.0.9001 is being loaded, but >= 1.0.0 is required

### ISSUE THREAD

**Issue by dkincaid** [rstudio/sparklyr#749]
While trying to install sparklyr using `devtools::install_github("rstudio/sparklyr")` I get the following error:

    Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
    namespace ‘dbplyr’ 0.0.0.9001 is being loaded, but >= 1.0.0 is required

I am unable to find dbplyr version >= 1.0.0

**Comment by ray-p144** [comment]
`dbplyr 1.0.0` is on CRAN.
Use this:
```r
install.packages("dbplyr", repos = "https://cloud.r-project.org")
```

**Comment by dkincaid** [comment]
I didn't realize dbplyr needed to be installed separately from dplyr and sparklyr. What is the proper order to load those three packages in order to make sure I get all the right versions?

**Comment by ray-p144** [comment]
I believe `install.packages` will install the packages that need to be imported by the package if they are available by default, but using `devtools::install_github` requires that those packages are already installed. 

If you want to check the packages that the current master branch `sparklyr` imports, they can be found [here](https://github.com/rstudio/sparklyr/blob/master/DESCRIPTION)

An example of what you can do since you want to install the most recent master version of the package is:
```r 
install.packages("dplyr", repos = "https://cloud.r-project.org")
install.packages("dbplyr", repos = "https://cloud.r-project.org")
devtools::install_github("rstudio/sparklyr")
```
You can also do this because using `install.packages` on `dbplyr` will load `dplyr`:
```r
install.packages("dbplyr", repos = "https://cloud.r-project.org")
devtools::install_github("rstudio/sparklyr")
```

**Comment by dkincaid** [comment]
Excellent! Thank you for the tips and helping get me past the error.

**Comment by javierluraschi** [comment]
@dkincaid `sparklyr` got patched as `0.5.6` to not require installing `dbplyr` manually at all. Closing, but let me know if you find out this is not the case.
